CITATION.cff

```diff
@@ -0,0 +1,35 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+title: "Triton Inference Server: an optimized cloud and edge inferencing solution."
+url: https://github.com/triton-inference-server
+repository-code: https://github.com/triton-inference-server/server
+authors:
+  - family-names: Goodwin
```

Comment:
For authors, we just need to add "NVIDIA Corporation". We shouldn't call out individual engineers. If this was a specific paper, that would be different, however, there are a lot of contributors. We should just be generic here.